### PR TITLE
Auditable logging app

### PR DESCRIPTION
**Pull request by takuro-sato**
Recreate PR https://github.com/microsoft/ccf-app-samples/pull/11, due to issue #43.

### TODOs for future
- [ ] Add demo like banking-app

### CODE REVIEW

File: auditable-logging-app/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+SHELL := /bin/bash
+
+.PHONY: help
+.DEFAULT_GOAL := help
+
+help: ## 💬 This help message :)
```
Comment (achamayou) [inline auditable-logging-app/Makefile:6]: First time I see emojis in Makefiles ^_^

File: auditable-logging-app/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Auditable logging application
```
[the remark this answers is not in the document]
Comment (achamayou) [reply auditable-logging-app/README.md:1]: That's fine, auditability is a generic property.

File: auditable-logging-app/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,3275 @@
+{
```
Comment (achamayou) [inline auditable-logging-app/package-lock.json:1]: Should that be checked in?

File: auditable-logging-app/src/endpoints/log.ts [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+
+interface Range {
+  start?: number;
+  last?: number;
+}
+
+type LogIdAccessType = "ANY" | "SPECIFIED_RANGE";
+interface LogIdAccess {
+  type: LogIdAccessType;
+  // Only for "PECIFIED_RANGE"
```
Comment (achamayou) [inline auditable-logging-app/src/endpoints/log.ts:54]:
```suggestion
  // Only for "SPECIFIED_RANGE"
```

**Review by achamayou — APPROVED** [review]

File: auditable-logging-app/Makefile (hunk 1 above)
Comment (takuro-sato) [reply auditable-logging-app/Makefile:6]: @ross-p-smith originally added it to banking-app. I like it :)

File: auditable-logging-app/package-lock.json (hunk 3 above)
Comment (takuro-sato) [reply auditable-logging-app/package-lock.json]: As we discussed, I deleted it.

**Pull request merged**